pkg/v4/resource/certconfig/create_test.go

```diff
@@ -55,6 +57,11 @@ func Test_ApplyCreateChange_Creates_createChange(t *testing.T) {
 		ToClusterGuestConfigFunc: func(v interface{}) (v1alpha1.ClusterGuestConfig, error) {
 			return v.(v1alpha1.ClusterGuestConfig), nil
 		},
+		ToClusterObjectMetaFunc: func(v interface{}) (apismetav1.ObjectMeta, error) {
```

Comment:
FYI there are meteadata accessors which are generic: https://github.com/giantswarm/operatorkit/blob/4df9ae8a194ea96785e9ed8fdd284d79904726be/controller/finalizer.go#L146

But this is also fine.

Reply:
I didn't know about that. I'll think about this for a second and decide. Thanks!

Reply:
I think this is a good idea, but requires quite amount of work in tests. I'll consider this once more and provider a spearate PR in case this is a good idea.